src/main/java/com/fryrank/controller/ReviewController.java

```diff
@@ -24,6 +29,16 @@ public GetAllReviewsOutput getAllReviewsForRestaurant(@RequestParam("restaurantI
         return reviewDAL.getAllReviewsByRestaurantId(restaurantId);
     }
 
+    @GetMapping(value = AGGREGATE_REVIEWS_URI)
+    public GetAggregateReviewInformationOutput getAggregateReviewInformationForRestaurants(
+            @RequestParam(name = "ids") String ids,
+            @RequestParam(name = "rating", defaultValue = "false") boolean includeRating
+    ) {
+        List<String> parsedIDs = Arrays.stream(ids.split(",")).sorted().collect(Collectors.toList());
```

Comment:
Minor: Consider sending a body with IDs as an array since there is extra work being done on front end to convert object -> list -> string when we could just send a list.

Reply:
Made a ticket https://github.com/NickPriv/FryRankBackend/issues/56